src/h5web/toolbar/LineToolbar.tsx

```diff
@@ -4,12 +4,16 @@ import ToggleBtn from './controls/ToggleBtn';
 import { useLineConfig } from '../vis-packs/core/line/config';
 import { CurveType } from '../vis-packs/core/line/models';
 import ToggleGroup from './controls/ToggleGroup';
-import Toolbar, { ToolbarProps } from './Toolbar';
+import Toolbar, { ToolbarControl } from './Toolbar';
 import Separator from './Separator';
 import ScaleSelector from './controls/ScaleSelector';
 
-function LineToolbar(props: ToolbarProps): ReactElement {
-  const { children } = props;
+interface Props {
+  errorControl?: ToolbarControl;
```

Comment:
This is so specific that it bothers me a bit. We could have a prop `additionnalControls?: ToolbarControl[] = []` ?

On the other hand, it is only used in `NxSpectrumToolbar`...

Reply:
Yeah, an array like `children` or `additionalControls` was too restrictive yet also too vague. If we ever use it in other places, then maybe we'll want to put controls in other places... If it comes down to that, we'll just duplicate the `LineToolbar`... For now, I thought it was clearer like this.